tests/datasets/test_datasets.py

```diff
@@ -91,3 +93,39 @@ def test_demo_replay_dataset():
     for _, actions in dataset:
         for action in actions:
             assert not action.has_option()
+
+
+def test_teacher_dataset():
+    """Test teacher dataset creation with Covers env.
+    """
+    # Test that data does not contain options since approach is random
+    utils.update_config({
+        "env": "cover",
+        "approach": "random",
+        "offline_data_method": "demo",
+        "offline_data_planning_timeout": 500,
+        "seed": 0,
```

Comment:
could you use seed 123 here instead of 0? sorry for the nit lol

explanation: i've been trying to do this across all our tests lol, the short explanation is that if we get in the habit of using seed 0 for our tests, then the tests that write out saved models will overwrite the saved models of our regular runs, since we often run `python src/main.py` with `--seed 0` out of habit

@tomsilver